docs/backup_and_restore.rst

```diff
@@ -123,67 +107,308 @@ Restoring
+   "Enter" to use the values that are populated for you. Proceed through the 
+   installation, finishing with ``./securedrop-admin tailsconfig.``
+
+   If SSH-over-Tor is configured, run ``ssh app`` and ``ssh mon`` to add the
+   new (temporary) onion URLs to your ``known_hosts`` file. 
+
+#. **Restore the backup**: Copy the backup archive (located in 
+   ``securedrop.old/install_files/ansible-base``) into the 
+   ``install_files/ansible-base`` directory and run
+
+   .. code:: sh
+            
+       ./securedrop-admin restore sd-backup-<your_backup_file>.tar.gz
+    
+   The restore task will proceed for some time, and then will fail with the
+   message ``ssh_exchange_identification: Connection closed by remote host``   
+   during the ``Wait for Tor to reload`` task. This is expected; during the 
+   restoration process, the *Application Server*'s onion URL changed, causing it
+   to be unreachable.
+
+   Reboot the *Application Server*, or log in via the console and issue the
+   command ``sudo service tor reload`` to restart the Tor service.
+
+#. **Obtain Application Server SSH .onion address**: If you are using 
+   SSH-over-Tor and are re-using the *Admin Workstation* 
+   from your previous installation, you will need to remove the SSH 
+   alias created during re-installation, or you will see a host key 
+   verification failure warning in the next section.  
+
+   First, obtain the SSH service onion address for the *Application Server.*
+
+   Inspect the ``app-ssh-aths`` file: 
+
+   .. code:: sh
+
+      cat install_files/ansible-base/app-ssh-aths
+
+   This file is in the format ``HidServAuth app_ssh_url Secret # client: admin``.
+   The ``app_ssh_url`` portion is your SSH service .onion address for the
+   *Application Server*.
+
+#. **Fix Tor Credentials:**
+   Type the following command, replacing ``<app_ssh_url>``: with the .onion
+   URL you obtained in the previous step.
+
+   .. code:: sh
+
+      ssh-keygen -f "/home/amnesia/.ssh/known_hosts" -R "<app_ssh_url>"
+
+#. **Replace original Tor credentials**: Copy the following files preserved 
+   from the old installation into
+   ``install_files/ansible-base``, overwriting the existing files of the 
+   same name. (If you wish, you may temporarily save the existing files
+   to another location outside of the SecureDrop directory before overwriting 
+   them).  
+
+   * ``app-source-ths``
+   * ``app-journalist-aths``
+   * ``app-ssh-aths``
+
+#. **Reconfigure Tor connections**: 
+   Run ``./securedrop-admin tailsconfig`` to update the *Admin Workstation*
+   to use the restored Onion URLs again, and verify connectivity to the 
+   servers and the *Source* and *Journalist Interfaces*. 
+   See :doc:`configure_admin_workstation_post_install`
+   for further instructions.
+
+#. **Delete temporary files**: Remove any temporary files or directories
+   you created as you performed this migration, including the 
+   ``securedrop.old`` directory.
+
```

Comment:
There's one other restore/migrate case to consider based on changes in https://github.com/freedomofpress/securedrop/pull/5677 - namely the v2+v3 -> v3 one (previously didn't work but now unblocked)

Reply:
See my comment- I have addressed this in a note in front of the v2+v3 instructions, indicating that migrating to Ubuntu 20.04 --> follow the v3-only instructions.